### PR TITLE
Bump to citus 11.0.0_beta

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: postgres-matrix.yml [hunk 1]
```diff
@@ -12,3 +12,5 @@ version_matrix:
       postgres_versions: [12, 13]
   - 10.2:
       postgres_versions: [12, 13, 14]
+  - 11.0:
```
Comment (onurctirtir) [inline postgres-matrix.yml:15]: really great to see that we have such a matrix :)

File: pkgvars [hunk 2]
```diff
@@ -1,4 +1,4 @@
 pkgname=citus
 pkgdesc='Citus (Open-Source)'
-pkglatest=10.2.5.citus-1
+pkglatest=11.0.0_beta.citus-1
```
Comment (onurctirtir) [inline pkgvars:3]: wondering should this also be `11.0.0-beta` ?

Comment (onurctirtir) [reply pkgvars:3]: nvm, it shouldn't

Comment (gurkanindibay) [reply pkgvars:3]: No our tag is 11.0.0_beta as well

**Review by onurctirtir — APPROVED** [review]

**Pull request merged**